Source/LogWriter.swift

```diff
@@ -141,10 +144,12 @@ open class OSLogWriter: LogModifierWriter {
     /// - Parameters:
     ///   - subsystem: The subsystem.
     ///   - category:  The category.
-    public init(subsystem: String, category: String, modifiers: [LogModifier] = []) {
+    public init(subsystem: String, category: String, modifiers: [LogModifier] = [], preserve: Bool = true) {
         self.subsystem = subsystem
         self.category = category
         self.modifiers = modifiers
+        self.preserve = preserve
+        self.format = preserve ? "%{public}@" :"%@"
```

Comment:
This changes current behavior btw - currently it was `%@` but because your preserve has default `true`, it will be `%{public}@` - not sure if it's expected.